webpack.config.js

```diff
@@ -21,18 +22,36 @@ export default {
     csvExtract: "./src/handlers/csv-extract/handler.ts",
     storeStandardisedInvoices:
       "./src/handlers/store-standardised-invoices/handler.ts",
+    frontend: "./src/handlers/frontend/handler.ts",
   },
   externals: "aws-sdk",
   mode: process.env.NODE_ENV === "dev" ? "development" : "production",
   module: {
-    rules: [{ test: /\.ts$/, use: "ts-loader", exclude: /node_modules/ }],
+    rules: [
+      { test: /.node$/, loader: "node-loader" },
+      { test: /\.ts$/, use: "ts-loader", exclude: /node_modules/ },
+    ],
   },
   output: {
     clean: true,
     filename: "[name].js",
     libraryTarget: "commonjs2",
     path: path.resolve(__dirname, "./dist"),
   },
+  plugins: [
+    new CopyPlugin({
+      patterns: [
+        {
+          from: "./node_modules/govuk-frontend",
+          to: "node_modules/govuk-frontend",
+        },
+        {
+          from: "./src/frontend/views",
+          to: "views",
+        },
+      ],
+    }),
+  ],
```

Comment:
can you explain these please?

Reply:
Ok, I didn’t see a views folder so I guess this doesn’t render anything as is? How come we need to copy the govuk frontend module specifically but not other dependencies?